README.md

```
@@ -1 +1,129 @@
+
+#### 변수 / 함수
+
+- **변수**: camelCase
+- **상수**: UPPER_SNAKE_CASE
+- **이벤트 핸들러**: `handle + 대상 + 동작` (예: `handleLoginSubmit`)
+
+## 팀원 소개
+
+|                                                           이진희                                                           |                                                          김영수                                                          |
+| :------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------: |
+| <a href="https://github.com/jjinheeWorld"><img src="https://avatars.githubusercontent.com/jjinheeWorld" width="100" /></a> | <a href="https://github.com/youngsoon12"><img src="https://avatars.githubusercontent.com/youngsoon12" width="100" /></a> |
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)